File: BEE 1013.java
import java.io.IOException;
import java.util.Scanner;

public class Main{
    public static void main(String[] args) throws IOException{
        Scanner input = new Scanner(System.in);
        int a = input.nextInt();
        int b = input.nextInt();
        int c = input.nextInt();
        int maiorAB = (a+b+Math.abs(a-b))/2;
        int greatestValue = (maiorAB+c+Math.abs(maiorAB-c))/2;
        System.out.println(greatestValue+" eh o maior");
    }
}

File: BEE 1013.cpp
#include<iostream>
#include<math.h>

using namespace std;

int main(){
    int a, b, c;
    cin >> a >> b >> c;
    int maiorAB = (a+b+abs(a-b))/2;
    int greatestValue = (maiorAB+c+abs(maiorAB-c))/2;
    cout<<greatestValue<<" eh o maior"<<endl;
    return 0;
}